week-5/A-dom-manipulation/exercise.js

```diff
@@ -4,30 +4,46 @@ Task 1
 
 Write JavaScript below that changes the background colour of the page when the 'Change colour' button is clicked.
 */
-
-
+var updateTitleBtn = document.querySelector("#bgrChangeBtn");
+updateTitleBtn.addEventListener("click", function() {
+  var myElement = document.querySelector("#bgrChangeBtn");
```

Comment:
This line looks identical to line 7, does that tell us something? We already have a reference to the button, so if we wanted to change that then we can just change it 🙂. We don't need to select it again within the event handler.

However it looks like the exercise asks you to change the background colour of the *page*, not the button when the button is clicked.